NUITizenGallery/res/layout/ScrollViewTest1Page.xaml

```diff
@@ -1,30 +1,32 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<base:View x:Class="NUITizenGallery.ScrollViewTest1Page"
+<comp:ContentPage x:Class="NUITizenGallery.ScrollViewTest1Page"
   xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
   xmlns:base="clr-namespace:Tizen.NUI.BaseComponents;assembly=Tizen.NUI"
   xmlns:comp="clr-namespace:Tizen.NUI.Components;assembly=Tizen.NUI.Components"
   xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+           WidthSpecification="-1"
+           HeightSpecification="-1"
            BackgroundColor="White"
-           PositionUsesPivotPoint="True" 
+           PositionUsesPivotPoint="True"
            ParentOrigin="Center"
-           PivotPoint="Center"          
-           WidthResizePolicy="FillToParent"
-           HeightResizePolicy="FillToParent">
+           PivotPoint="Center">
 
-    <base:View x:Name="rootView" HeightResizePolicy="FillToParent" >
+    <comp:ContentPage.AppBar>
+        <comp:AppBar x:Name="appBar" Title="ScrollViewTest1"/>
+    </comp:ContentPage.AppBar>
 
-        <base:View.Layout>
-            <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="10, 10"/>
-        </base:View.Layout>
-
-        <base:View x:Name="lblView">
-            <comp:ScrollableBase x:Name="Scroller" ScrollingDirection="Vertical" HideScrollbar="false" >
+    <comp:ContentPage.Content>
+        <base:View x:Name="lblView" WidthSpecification="-1" HeightSpecification="-1">
+            <base:View.Layout>
+                <LinearLayout LinearOrientation="Vertical" LinearAlignment="Begin" CellPadding="10, 10"/>
+            </base:View.Layout>
+            <comp:Button x:Name="btn" Text="Test ScrollTo" WidthSpecification="{Static LayoutParamPolicies.MatchParent}" HeightSpecification="100"/>
+            <comp:ScrollableBase x:Name="Scroller" ScrollingDirection="Vertical" HideScrollbar="false" WidthSpecification="-1" HeightSpecification="-1">

```

Comment:
use "{Static LayoutParamPolicies.MatchParent}" not -1

Reply:
Changed.